tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/cvat_hands_and_palm.py

```diff
@@ -177,31 +186,31 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
             if progress_callback is not None and ann_id % progress_interval == 0:
                 progress_callback(ann_id * 100 / len(annotation['annotations']))
 
-        return ConverterReturn(annotations, meta, content_errors)
+        return ConverterReturn(annotations, {'label_map': {0: 'PALM'}, 'label': ['PALM']}, content_errors)
```

Comment:
why meta becomes not enough here?

Reply:
Meta returned from get_meta() contains information about all categories in dataset, no difference for hand landmarks and palm detection annotations. In dataset category PALM corresponds to index (label) 22. Palm detection model predicts only one class and metric expects annotation with label 0.

Reply:
do we need whole meta or we can just prepare it for running?